README.md

```diff
@@ -1,24 +1,21 @@
-# GitHub Action: 42Crunch REST API Static Security Testing
+# GitHub Action: 42Crunch Free Version - REST API Static Security Testing
 
 The REST API Static Security Testing action locates REST API contracts that follow the OpenAPI Specification (OAS, formerly known as Swagger) and runs thorough security checks on them. Both OAS v2 and v3.0.x are supported, in both JSON and YAML format.
 
-You can use this action in the following scenarios:
+This Github action is working in freemium mode: organizations can run 25 audits per month per repository, with a maximum of 3 repositories per organisation.
 
-- Add automatic static application security testing (SAST) task to your CI/CD workflows.
+You can use this action in the following scenarios:
+- Add automatic static API security testing (SAST) task to your CI/CD workflows.
 - Perform these checks on pull request reviews and/or code merges.
 - Flag the located issues in GitHub's Security / Code Scanning Alerts.
 
 ## Discover APIs in your repositories
 
 By default, this action will:
 
-1. Look for any `.json` and `.yaml` files in the repository.
-2. Pick the files that use OpenAPI schema.
-3. Perform security audit on the OpenAPI definitions.
-
-This way, you can locate any new or changed API contracts in the repository.
-
-All discovered APIs are uploaded to an API collection in 42Crunch Platform. The action uses the environment variables `GITHUB_REPOSITORY` and `GITHUB_REF` to name the repository and the branch/tag/PR name from where the API collection originated from. During the subsequent action runs, the APIs in the collection are kept in sync with the changes in your repository.
+1. Look for any `.json` and `.yaml/.yml` files in the repository.
+2. Pick the files that use OpenAPI/Swagger 2.0/3.0x schemas.
```

Comment:
This is problematic, because the order in "OpenAPI/Swagger" does not match the order of `2.0/3.0.x`, and I don't really see an elegant solution to align them (either you'd be promoting the wrong name, or the versions would be in illogical order).

Since it was called  "the OpenAPI Specification" already during v2.0, I'd recommend using just that to refer to the standard here: "... files that use the OAS v2.0 or 3.0x schemas"